src/item.cpp

```diff
@@ -1625,7 +1625,7 @@ nc_color item::color_in_inventory() const
         ret = c_cyan;
     } else if(has_flag("LITCIG")) {
         ret = c_red;
-    } else if ( has_flag("LEAK_DAM") && has_flag("RADIOACTIVE") && damage > 0 ) {
+    } else if ( (has_flag("LEAK_DAM") || has_flag("LEAK_DAM_SMALL")) && has_flag("RADIOACTIVE") && damage > 0 ) {
```

Comment:
I don't like to be that person, but:

The flag appears to be new. Where is it documented?